sdk/networkcloud/Azure.ResourceManager.NetworkCloud/src/Generated/NetworkCloudVirtualMachineResource.cs

```diff
@@ -366,16 +366,16 @@ public virtual ArmOperation<NetworkCloudVirtualMachineResource> Update(WaitUntil
         /// <param name="waitUntil"> <see cref="WaitUntil.Completed"/> if the method should wait to return until the long-running operation has completed on the service; <see cref="WaitUntil.Started"/> if it should return after starting the operation. For more information on long-running operations, please see <see href="https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md"> Azure.Core Long-Running Operation samples</see>. </param>
         /// <param name="content"> The request body. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual async Task<ArmOperation> PowerOffAsync(WaitUntil waitUntil, VirtualMachinePowerOffContent content = null, CancellationToken cancellationToken = default)
+        public virtual async Task<ArmOperation<OperationStatusResult>> PowerOffAsync(WaitUntil waitUntil, VirtualMachinePowerOffContent content = null, CancellationToken cancellationToken = default)
```

Comment:
this `OperationStatusResult` looks like an internal type for the LRO, do we have to expose this in the public API?

Reply:
That's the point of `final-state-schema`, user wants to expose the type

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/b5f54222a4a925eb0239cd6653ab8b27185195f4/specification/networkcloud/resource-manager/Microsoft.NetworkCloud/stable/2023-07-01/networkcloud.json#L5588